csharp/events/Program.cs

```diff
@@ -96,7 +96,7 @@ public void Search(string directory, string searchPattern, bool searchSubDirs =
                 {
                     directoryChanged?.Invoke(this,
                         new SearchDirectoryArgs(dir, totalDirs, completedDirs++));
-                    // Recursively search this child directory:
+                    // Search each sub-directory for files that match the search pattern:
```

Comment:
Perhaps "Search 'dir' and its  subdirectories for files that match the search pattern."?